### PR TITLE
M2-5798: Dataviz - Slider Rows Item support

### PR DESCRIPTION
**Pull request by nmialik**
- [x] Tests for the changes have been added
- [x] Delivered the `fix` or `feature` branches into `develop` or `release` branches via `Squash and Merge` (to keep clean history)

### 📝 Description

🔗 [Jira Ticket M2-5798](https://mindlogger.atlassian.net/browse/M2-5798)

### 📸 Screenshots

| Before (Optional)                      | After                                 |
| -------------------------------------- | ------------------------------------- |
|Снимок экрана 2024-04-02 в 16 07 09 | Снимок экрана 2024-04-02 в 16 06 22
 |Снимок экрана 2024-04-02 в 16 07 09 | Снимок экрана 2024-04-02 в 16 06 40

### CODE REVIEW

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts [hunk 1]
```diff
@@ -79,6 +81,12 @@ export type SingleMultiSelectPerRowItemAnswer = {
   'data-testid'?: string;
 };
 
+export type SliderRowsItemAnswer = {
```
Comment (alex-i-e) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts]: Does it make sense to generate new type via generic `CreateItemAnswer` here and above in `...ItemAnswer` types?

Comment (nmialik) [reply src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts]: So there are different types, in the first case - formatted data for the summary tab, and in here - simply decrypted data for the response tab

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Review/Review.utils.tsx [hunk 2]
```diff
@@ -95,5 +97,7 @@ export const getResponseItem = (activityItemAnswer: ActivityItemAnswer) => {
           {...(activityItemAnswer as SingleMultiSelectPerRowItemAnswer)}
         />
       );
+    case ItemResponseType.SliderRows:
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Review/Review.utils.tsx:100]: Please add this case to the Review.utils.test.ts

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts (hunk 1 above)
Comment (alex-i-e) [reply src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts]: I meant another generic type, different but that looks like `CreateFormattedReposense`

Comment (nmialik) [reply src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types.ts]: done

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by alex-i-e — APPROVED** [review]

**Pull request merged**